stable/datadog/values.yaml

```diff
@@ -174,16 +174,6 @@ datadog:
   #
   # logsConfigContainerCollectAll: false
 
-  ## @param apm - object - required
-  ## Enable apm agent and provide custom configs
-  #
-  apm:
-    ## @param enabled - boolean - optional - default: false
-    ## Enable this to enable APM and tracing, on port 8126
-    ## ref: https://github.com/DataDog/docker-dd-agent#tracing-from-the-host
-    #
-    enabled: false
```

Comment:
I think it is better to activate it by default, but still give the possibility to user to deactivate it.